<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="37" passed="10" failed="0" skipped="25">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-12-22T10:45:52 IST" name="Smoke" finished-at="2023-12-22T10:47:00 IST" duration-ms="67603">
    <groups>
    </groups>
    <test started-at="2023-12-22T10:45:52 IST" name="LoginTest" finished-at="2023-12-22T10:46:06 IST" duration-ms="14189">
      <class name="Tests.LoginTest">
        <test-method signature="TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method)[pri:1, instance:Tests.LoginTest@c9413d8]" started-at="2023-12-22T10:45:52 IST" name="TC_001_VerifyLogin" description="This  test case verifies the login functionality of the application using the provided test data." data-provider="Testdatas" finished-at="2023-12-22T10:45:52 IST" duration-ms="9" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Mobilenumber=8248606917, Username=vishaldel, Password=vishalde, Runmode=N, Project=jj_test, TC ID=TC_001}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.LoginTest.TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test case as the Run mode for data set is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test case as the Run mode for data set is NO
at Utilities.Utils.checkexecution(Utils.java:225)
at Tests.LoginTest.TC_001_VerifyLogin(LoginTest.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyLogin -->
        <test-method signature="TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method)[pri:1, instance:Tests.LoginTest@c9413d8]" started-at="2023-12-22T10:45:52 IST" name="TC_001_VerifyLogin" description="This  test case verifies the login functionality of the application using the provided test data." data-provider="Testdatas" finished-at="2023-12-22T10:45:52 IST" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Mobilenumber=8248606917, Username=vishalde, Password=vishaldel, Runmode=N, Project=jj_test, TC ID=TC_002}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.LoginTest.TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test case as the Run mode for data set is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test case as the Run mode for data set is NO
at Utilities.Utils.checkexecution(Utils.java:225)
at Tests.LoginTest.TC_001_VerifyLogin(LoginTest.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyLogin -->
        <test-method signature="TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method)[pri:1, instance:Tests.LoginTest@c9413d8]" started-at="2023-12-22T10:45:52 IST" name="TC_001_VerifyLogin" description="This  test case verifies the login functionality of the application using the provided test data." data-provider="Testdatas" finished-at="2023-12-22T10:45:52 IST" duration-ms="1" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Mobilenumber=8248606917, Username=vishaldel, Password=vishaldel, Runmode=N, Project=jj_tes, TC ID=TC_003}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.LoginTest.TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test case as the Run mode for data set is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test case as the Run mode for data set is NO
at Utilities.Utils.checkexecution(Utils.java:225)
at Tests.LoginTest.TC_001_VerifyLogin(LoginTest.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyLogin -->
        <test-method signature="TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method)[pri:1, instance:Tests.LoginTest@c9413d8]" started-at="2023-12-22T10:45:52 IST" name="TC_001_VerifyLogin" description="This  test case verifies the login functionality of the application using the provided test data." data-provider="Testdatas" finished-at="2023-12-22T10:45:52 IST" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Mobilenumber=8248606917, Username=vishalde, Password=vishalde, Runmode=N, Project=jj_tes, TC ID=TC_004}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.LoginTest.TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test case as the Run mode for data set is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test case as the Run mode for data set is NO
at Utilities.Utils.checkexecution(Utils.java:225)
at Tests.LoginTest.TC_001_VerifyLogin(LoginTest.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyLogin -->
        <test-method signature="TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method)[pri:1, instance:Tests.LoginTest@c9413d8]" started-at="2023-12-22T10:45:52 IST" name="TC_001_VerifyLogin" description="This  test case verifies the login functionality of the application using the provided test data." data-provider="Testdatas" finished-at="2023-12-22T10:45:52 IST" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Mobilenumber=8248606917, Username=, Password=vishaldel, Runmode=N, Project=jj_test, TC ID=TC_005}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.LoginTest.TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test case as the Run mode for data set is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test case as the Run mode for data set is NO
at Utilities.Utils.checkexecution(Utils.java:225)
at Tests.LoginTest.TC_001_VerifyLogin(LoginTest.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyLogin -->
        <test-method signature="TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method)[pri:1, instance:Tests.LoginTest@c9413d8]" started-at="2023-12-22T10:45:52 IST" name="TC_001_VerifyLogin" description="This  test case verifies the login functionality of the application using the provided test data." data-provider="Testdatas" finished-at="2023-12-22T10:45:52 IST" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Mobilenumber=8248606917, Username=vishaldel, Password=, Runmode=N, Project=jj_test, TC ID=TC_006}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.LoginTest.TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test case as the Run mode for data set is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test case as the Run mode for data set is NO
at Utilities.Utils.checkexecution(Utils.java:225)
at Tests.LoginTest.TC_001_VerifyLogin(LoginTest.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyLogin -->
        <test-method signature="TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method)[pri:1, instance:Tests.LoginTest@c9413d8]" started-at="2023-12-22T10:45:52 IST" name="TC_001_VerifyLogin" description="This  test case verifies the login functionality of the application using the provided test data." data-provider="Testdatas" finished-at="2023-12-22T10:45:52 IST" duration-ms="1" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Mobilenumber=8248606917, Username=vishaldel, Password=vishaldel, Runmode=N, Project=, TC ID=TC_007}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.LoginTest.TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test case as the Run mode for data set is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test case as the Run mode for data set is NO
at Utilities.Utils.checkexecution(Utils.java:225)
at Tests.LoginTest.TC_001_VerifyLogin(LoginTest.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyLogin -->
        <test-method signature="TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method)[pri:1, instance:Tests.LoginTest@c9413d8]" started-at="2023-12-22T10:45:52 IST" name="TC_001_VerifyLogin" description="This  test case verifies the login functionality of the application using the provided test data." data-provider="Testdatas" finished-at="2023-12-22T10:45:52 IST" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Mobilenumber=, Username=vishaldel, Password=vishaldel, Runmode=N, Project=jj_test, TC ID=TC_008}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.LoginTest.TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test case as the Run mode for data set is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test case as the Run mode for data set is NO
at Utilities.Utils.checkexecution(Utils.java:225)
at Tests.LoginTest.TC_001_VerifyLogin(LoginTest.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyLogin -->
        <test-method signature="TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method)[pri:1, instance:Tests.LoginTest@c9413d8]" started-at="2023-12-22T10:45:52 IST" name="TC_001_VerifyLogin" description="This  test case verifies the login functionality of the application using the provided test data." data-provider="Testdatas" finished-at="2023-12-22T10:45:52 IST" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Mobilenumber=, Username=, Password=, Runmode=N, Project=, TC ID=TC_009}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.LoginTest.TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test case as the Run mode for data set is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test case as the Run mode for data set is NO
at Utilities.Utils.checkexecution(Utils.java:225)
at Tests.LoginTest.TC_001_VerifyLogin(LoginTest.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyLogin -->
        <test-method signature="TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method)[pri:1, instance:Tests.LoginTest@c9413d8]" started-at="2023-12-22T10:45:52 IST" name="TC_001_VerifyLogin" description="This  test case verifies the login functionality of the application using the provided test data." data-provider="Testdatas" finished-at="2023-12-22T10:46:06 IST" duration-ms="14088" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Mobilenumber=8248606917, Username=E1M4, Password=E1M4, Runmode=Y, Project=framedemo_d1, TC ID=TC_010}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.LoginTest.TC_001_VerifyLogin(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyLogin -->
      </class> <!-- Tests.LoginTest -->
    </test> <!-- LoginTest -->
    <test started-at="2023-12-22T10:46:07 IST" name="VersionTest" finished-at="2023-12-22T10:46:07 IST" duration-ms="6">
      <class name="Tests.LoginTest">
        <test-method signature="TC_002_Verify_AppVersion(java.util.Hashtable,java.lang.reflect.Method)[pri:2, instance:Tests.LoginTest@7eb01b12]" started-at="2023-12-22T10:46:07 IST" name="TC_002_Verify_AppVersion" description="This  test case verifies the version of the application." data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, Version=V 3.1.3, TC ID=TC_012}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.LoginTest.TC_002_Verify_AppVersion(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_002_VERIFY_APPVERSION as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_002_VERIFY_APPVERSION as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.LoginTest.TC_002_Verify_AppVersion(LoginTest.java:29)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_Verify_AppVersion -->
      </class> <!-- Tests.LoginTest -->
    </test> <!-- VersionTest -->
    <test started-at="2023-12-22T10:46:07 IST" name="AttendanceTest" finished-at="2023-12-22T10:46:07 IST" duration-ms="63">
      <class name="Tests.AttendanceTest">
        <test-method signature="TC_001_VerifyAttendance(java.util.Hashtable,java.lang.reflect.Method)[pri:3, instance:Tests.AttendanceTest@43ed0ff3]" started-at="2023-12-22T10:46:07 IST" name="TC_001_VerifyAttendance" description="This  test case verifies the Attendance is submit or not" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="3" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, Image Required=true, Attendancetype=Present, TC ID=TC_004}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.AttendanceTest.TC_001_VerifyAttendance(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_001_VERIFYATTENDANCE as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_001_VERIFYATTENDANCE as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.AttendanceTest.TC_001_VerifyAttendance(AttendanceTest.java:19)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyAttendance -->
        <test-method signature="TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method)[pri:4, instance:Tests.AttendanceTest@43ed0ff3]" started-at="2023-12-22T10:46:07 IST" name="TC_002_VerifyImagerequired" description="This  test case verifies the attendance images is mandatory or not for each types" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="1" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, Attendancetype=Present, TC ID=TC_003}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.AttendanceTest.TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.AttendanceTest.TC_002_VerifyImagerequired(AttendanceTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_VerifyImagerequired -->
        <test-method signature="TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method)[pri:4, instance:Tests.AttendanceTest@43ed0ff3]" started-at="2023-12-22T10:46:07 IST" name="TC_002_VerifyImagerequired" description="This  test case verifies the attendance images is mandatory or not for each types" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, Attendancetype=Leave, TC ID=TC_003}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.AttendanceTest.TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.AttendanceTest.TC_002_VerifyImagerequired(AttendanceTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_VerifyImagerequired -->
        <test-method signature="TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method)[pri:4, instance:Tests.AttendanceTest@43ed0ff3]" started-at="2023-12-22T10:46:07 IST" name="TC_002_VerifyImagerequired" description="This  test case verifies the attendance images is mandatory or not for each types" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="1" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, Attendancetype=Absent, TC ID=TC_003}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.AttendanceTest.TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.AttendanceTest.TC_002_VerifyImagerequired(AttendanceTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_VerifyImagerequired -->
        <test-method signature="TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method)[pri:4, instance:Tests.AttendanceTest@43ed0ff3]" started-at="2023-12-22T10:46:07 IST" name="TC_002_VerifyImagerequired" description="This  test case verifies the attendance images is mandatory or not for each types" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, Attendancetype=At Office, TC ID=TC_003}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.AttendanceTest.TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.AttendanceTest.TC_002_VerifyImagerequired(AttendanceTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_VerifyImagerequired -->
        <test-method signature="TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method)[pri:4, instance:Tests.AttendanceTest@43ed0ff3]" started-at="2023-12-22T10:46:07 IST" name="TC_002_VerifyImagerequired" description="This  test case verifies the attendance images is mandatory or not for each types" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="1" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, Attendancetype=Week off, TC ID=TC_003}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.AttendanceTest.TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.AttendanceTest.TC_002_VerifyImagerequired(AttendanceTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_VerifyImagerequired -->
        <test-method signature="TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method)[pri:4, instance:Tests.AttendanceTest@43ed0ff3]" started-at="2023-12-22T10:46:07 IST" name="TC_002_VerifyImagerequired" description="This  test case verifies the attendance images is mandatory or not for each types" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, Attendancetype=Holiday, TC ID=TC_003}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.AttendanceTest.TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.AttendanceTest.TC_002_VerifyImagerequired(AttendanceTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_VerifyImagerequired -->
        <test-method signature="TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method)[pri:4, instance:Tests.AttendanceTest@43ed0ff3]" started-at="2023-12-22T10:46:07 IST" name="TC_002_VerifyImagerequired" description="This  test case verifies the attendance images is mandatory or not for each types" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="3" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, Attendancetype=Tour, TC ID=TC_003}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.AttendanceTest.TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.AttendanceTest.TC_002_VerifyImagerequired(AttendanceTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_VerifyImagerequired -->
        <test-method signature="TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method)[pri:4, instance:Tests.AttendanceTest@43ed0ff3]" started-at="2023-12-22T10:46:07 IST" name="TC_002_VerifyImagerequired" description="This  test case verifies the attendance images is mandatory or not for each types" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="3" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=N, Attendancetype=Resign, TC ID=TC_003}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.AttendanceTest.TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.AttendanceTest.TC_002_VerifyImagerequired(AttendanceTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_VerifyImagerequired -->
        <test-method signature="TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method)[pri:4, instance:Tests.AttendanceTest@43ed0ff3]" started-at="2023-12-22T10:46:07 IST" name="TC_002_VerifyImagerequired" description="This  test case verifies the attendance images is mandatory or not for each types" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="9" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, Attendancetype=Training, TC ID=TC_003}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.AttendanceTest.TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.AttendanceTest.TC_002_VerifyImagerequired(AttendanceTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_VerifyImagerequired -->
        <test-method signature="TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method)[pri:4, instance:Tests.AttendanceTest@43ed0ff3]" started-at="2023-12-22T10:46:07 IST" name="TC_002_VerifyImagerequired" description="This  test case verifies the attendance images is mandatory or not for each types" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="3" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, Attendancetype=Monthly Meeting, TC ID=TC_003}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.AttendanceTest.TC_002_VerifyImagerequired(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_002_VERIFYIMAGEREQUIRED as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.AttendanceTest.TC_002_VerifyImagerequired(AttendanceTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_002_VerifyImagerequired -->
      </class> <!-- Tests.AttendanceTest -->
    </test> <!-- AttendanceTest -->
    <test started-at="2023-12-22T10:46:07 IST" name="DownloadcallsTest" finished-at="2023-12-22T10:46:07 IST" duration-ms="18">
      <class name="Tests.DownloadcallsTest">
        <test-method signature="TC_001_VerifyDownloadcalls(java.util.Hashtable,java.lang.reflect.Method)[pri:5, instance:Tests.DownloadcallsTest@5c1bd44c]" started-at="2023-12-22T10:46:07 IST" name="TC_001_VerifyDownloadcalls" description="This  test case verifies the calls is downloaded or not in download calls" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="1" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{TargetId=33011, Runmode=Y, TC ID=TC_001}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.DownloadcallsTest.TC_001_VerifyDownloadcalls(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_001_VERIFYDOWNLOADCALLS as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_001_VERIFYDOWNLOADCALLS as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.DownloadcallsTest.TC_001_VerifyDownloadcalls(DownloadcallsTest.java:18)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyDownloadcalls -->
        <test-method signature="TC_001_VerifyDownloadcalls(java.util.Hashtable,java.lang.reflect.Method)[pri:5, instance:Tests.DownloadcallsTest@5c1bd44c]" started-at="2023-12-22T10:46:07 IST" name="TC_001_VerifyDownloadcalls" description="This  test case verifies the calls is downloaded or not in download calls" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{TargetId=000000, Runmode=Y, TC ID=TC_002}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.DownloadcallsTest.TC_001_VerifyDownloadcalls(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_001_VERIFYDOWNLOADCALLS as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_001_VERIFYDOWNLOADCALLS as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.DownloadcallsTest.TC_001_VerifyDownloadcalls(DownloadcallsTest.java:18)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyDownloadcalls -->
        <test-method signature="TC_001_VerifyDownloadcalls(java.util.Hashtable,java.lang.reflect.Method)[pri:5, instance:Tests.DownloadcallsTest@5c1bd44c]" started-at="2023-12-22T10:46:07 IST" name="TC_001_VerifyDownloadcalls" description="This  test case verifies the calls is downloaded or not in download calls" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{TargetId=33011, Runmode=Y, TC ID=TC_003}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.DownloadcallsTest.TC_001_VerifyDownloadcalls(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_001_VERIFYDOWNLOADCALLS as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_001_VERIFYDOWNLOADCALLS as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.DownloadcallsTest.TC_001_VerifyDownloadcalls(DownloadcallsTest.java:18)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyDownloadcalls -->
        <test-method signature="TC_001_VerifyDownloadcalls(java.util.Hashtable,java.lang.reflect.Method)[pri:5, instance:Tests.DownloadcallsTest@5c1bd44c]" started-at="2023-12-22T10:46:07 IST" name="TC_001_VerifyDownloadcalls" description="This  test case verifies the calls is downloaded or not in download calls" data-provider="Testdatas" finished-at="2023-12-22T10:46:07 IST" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{TargetId=, Runmode=, TC ID=}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.DownloadcallsTest.TC_001_VerifyDownloadcalls(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test TC_001_VERIFYDOWNLOADCALLS as the Run mode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test TC_001_VERIFYDOWNLOADCALLS as the Run mode is NO
at Utilities.Utils.checkexecution(Utils.java:220)
at Tests.DownloadcallsTest.TC_001_VerifyDownloadcalls(DownloadcallsTest.java:18)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_001_VerifyDownloadcalls -->
      </class> <!-- Tests.DownloadcallsTest -->
    </test> <!-- DownloadcallsTest -->
    <test started-at="2023-12-22T10:46:07 IST" name="ResourceCentre Test" finished-at="2023-12-22T10:47:00 IST" duration-ms="52717">
      <class name="Tests.ResourceCentreTest">
        <test-method signature="TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method)[pri:5, instance:Tests.ResourceCentreTest@6e57e95e]" started-at="2023-12-22T10:46:07 IST" name="TC001_VerifyResourcecentrefiles" description="This  test case verifies the all type of extension files is downloaded or not ." data-provider="Testdatas" finished-at="2023-12-22T10:46:14 IST" duration-ms="6941" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Filename=Lead 2023 design.pptx, Runmode=Y, TC ID=TC_001}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.ResourceCentreTest.TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC001_VerifyResourcecentrefiles -->
        <test-method signature="TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method)[pri:5, instance:Tests.ResourceCentreTest@6e57e95e]" started-at="2023-12-22T10:46:14 IST" name="TC001_VerifyResourcecentrefiles" description="This  test case verifies the all type of extension files is downloaded or not ." data-provider="Testdatas" finished-at="2023-12-22T10:46:19 IST" duration-ms="4809" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Filename=Mobile App Toolkit_free.mp4, Runmode=Y, TC ID=TC_002}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.ResourceCentreTest.TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC001_VerifyResourcecentrefiles -->
        <test-method signature="TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method)[pri:5, instance:Tests.ResourceCentreTest@6e57e95e]" started-at="2023-12-22T10:46:19 IST" name="TC001_VerifyResourcecentrefiles" description="This  test case verifies the all type of extension files is downloaded or not ." data-provider="Testdatas" finished-at="2023-12-22T10:46:25 IST" duration-ms="6102" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Filename=NOGST_NOPAN.pdf, Runmode=Y, TC ID=TC_003}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.ResourceCentreTest.TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC001_VerifyResourcecentrefiles -->
        <test-method signature="TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method)[pri:5, instance:Tests.ResourceCentreTest@6e57e95e]" started-at="2023-12-22T10:46:25 IST" name="TC001_VerifyResourcecentrefiles" description="This  test case verifies the all type of extension files is downloaded or not ." data-provider="Testdatas" finished-at="2023-12-22T10:46:30 IST" duration-ms="5574" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Filename=NOGST_PAN.pdf, Runmode=Y, TC ID=TC_004}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.ResourceCentreTest.TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC001_VerifyResourcecentrefiles -->
        <test-method signature="TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method)[pri:5, instance:Tests.ResourceCentreTest@6e57e95e]" started-at="2023-12-22T10:46:30 IST" name="TC001_VerifyResourcecentrefiles" description="This  test case verifies the all type of extension files is downloaded or not ." data-provider="Testdatas" finished-at="2023-12-22T10:46:36 IST" duration-ms="5616" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Filename=redipae.docx, Runmode=Y, TC ID=TC_005}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.ResourceCentreTest.TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC001_VerifyResourcecentrefiles -->
        <test-method signature="TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method)[pri:5, instance:Tests.ResourceCentreTest@6e57e95e]" started-at="2023-12-22T10:46:36 IST" name="TC001_VerifyResourcecentrefiles" description="This  test case verifies the all type of extension files is downloaded or not ." data-provider="Testdatas" finished-at="2023-12-22T10:46:41 IST" duration-ms="5265" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Filename=retail-industry-trends.jpg, Runmode=Y, TC ID=TC_006}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.ResourceCentreTest.TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC001_VerifyResourcecentrefiles -->
        <test-method signature="TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method)[pri:5, instance:Tests.ResourceCentreTest@6e57e95e]" started-at="2023-12-22T10:46:41 IST" name="TC001_VerifyResourcecentrefiles" description="This  test case verifies the all type of extension files is downloaded or not ." data-provider="Testdatas" finished-at="2023-12-22T10:46:48 IST" duration-ms="6845" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Filename=Task Target Mapping (29)(1).xlsx, Runmode=Y, TC ID=TC_007}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.ResourceCentreTest.TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC001_VerifyResourcecentrefiles -->
        <test-method signature="TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method)[pri:5, instance:Tests.ResourceCentreTest@6e57e95e]" started-at="2023-12-22T10:46:48 IST" name="TC001_VerifyResourcecentrefiles" description="This  test case verifies the all type of extension files is downloaded or not ." data-provider="Testdatas" finished-at="2023-12-22T10:46:53 IST" duration-ms="4825" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Filename=Tech-Trends-in-Retail.jpg, Runmode=Y, TC ID=TC_008}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.ResourceCentreTest.TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC001_VerifyResourcecentrefiles -->
        <test-method signature="TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method)[pri:5, instance:Tests.ResourceCentreTest@6e57e95e]" started-at="2023-12-22T10:46:53 IST" name="TC001_VerifyResourcecentrefiles" description="This  test case verifies the all type of extension files is downloaded or not ." data-provider="Testdatas" finished-at="2023-12-22T10:47:00 IST" duration-ms="6722" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Filename=The Merchandising black book.pptx, Runmode=Y, TC ID=TC_009}]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public static void Tests.ResourceCentreTest.TC001_VerifyResourcecentrefiles(java.util.Hashtable,java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC001_VerifyResourcecentrefiles -->
      </class> <!-- Tests.ResourceCentreTest -->
      <class name="Tests.LoginTest">
        <test-method is-config="true" signature="StartApp(org.testng.ITestContext)[pri:0, instance:Tests.LoginTest@7eb01b12]" started-at="2023-12-22T10:45:35 IST" name="StartApp" finished-at="2023-12-22T10:45:52 IST" duration-ms="16733" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@195113de]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StartApp -->
        <test-method is-config="true" signature="tearDownApp()[pri:0, instance:Tests.LoginTest@7eb01b12]" started-at="2023-12-22T10:47:00 IST" name="tearDownApp" finished-at="2023-12-22T10:47:01 IST" duration-ms="1365" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownApp -->
      </class> <!-- Tests.LoginTest -->
    </test> <!-- ResourceCentre Test -->
  </suite> <!-- Smoke -->
</testng-results>
